### PR TITLE
fix installing-controlplane-on-k8s.md

### PR DESCRIPTION
**Pull request by jz543fm**
**What this PR does / why we need it**:
Updates docs how to Install Control Plane on Kubernetes for (MySQL + MINIO)

**Which issue(s) this PR fixes**:
Issue [4363](https://github.com/pipe-cd/pipecd/issues/4363)

Fixes #

**Does this PR introduce a user-facing change?**:

- **How are users affected by this change**:
- **Is this breaking change**:
- **How to migrate (if breaking change)**:

### CODE REVIEW

**Comment by jz543fm** [comment]
I do not know why DCO check does not work, I commit with `git add *; git commit -s -am "fix installing-controlplane-on-k8s.md"; git push`

**Comment by t-kikuc** [comment]
@jz543fm 
Sorry, it seems the DCO checker's problem...
We'll investigate it.

**Comment by khanhtc1202** [comment]
@jz543fm Thanks for the PR, I will check the DCO stage, looks like it blocks other PRs as well 🏃

**Comment by jz543fm** [comment]
Ok, I will wait for fix, yea, that's why I closed the first PR...

File: docs/content/en/docs-v0.46.x/installation/install-control-plane/installing-controlplane-on-k8s.md [hunk 1]
```diff
@@ -72,7 +74,7 @@ helm upgrade -i pipecd oci://ghcr.io/pipe-cd/chart/pipecd --version {{< blocks/l
   --set-file secret.gcsServiceAccount.data=path-to-service-account-file
 ```
 
-Currently, besides `Firestore` PipeCD supports other databases as its datastore such as `MySQL`. Also as for filestore, PipeCD supports `AWS S3` and `MINIO` either.
+#### Using MySQL and MINIO
```
Comment (khanhtc1202) [inline docs/content/en/docs-v0.46.x/installation/install-control-plane/installing-controlplane-on-k8s.md:77]: How about using the tab switchable configuration like this?
ref: https://github.com/pipe-cd/pipecd/blob/master/docs/content/en/docs-dev/installation/install-piped/installing-on-kubernetes.md?plain=1#L46-L86

https://github.com/pipe-cd/pipecd/assets/32532742/0270e01c-a246-4965-88ea-9060db6c2bdb

Comment (khanhtc1202) [reply docs/content/en/docs-v0.46.x/installation/install-control-plane/installing-controlplane-on-k8s.md:77]: You can see how it looks here: https://pipecd.dev/docs-v0.47.x/installation/install-piped/installing-on-kubernetes/#:~:text=the%20full%20configuration.-,Installing%20by%20using,-Helm%20(3.8.0%20or

**Comment by jz543fm** [comment]
@khanhtc1202 fixed

**Review by t-kikuc — APPROVED** [review]
Thanks!

We should copy the change into v0.47.x and dev.

**Review by khanhtc1202 — APPROVED** [review]
LGTM, thank you 🙌

**Pull request merged**